catalog-info.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: contentful-management.js
+  description: |
+    Contentful command line interface. Use Contentful features straight from the command line.
```

Comment:
I think that description is wrong. As a default, I suggest we go with the repo description.

Reply:
true, I was copying it for all from contentful cli, must have missed to change it here